src/main/features/dashboard/views/macro/claimStatus/claimFullyAdmitted.njk

```diff
@@ -0,0 +1,115 @@
+    {% set paymentSchedule = repaymentPlan.paymentSchedule | renderPaymentSchedule %}
+
+    <p> {{ t('You’ve offered to pay {{ claimantName }} {{ amount }} {{ paymentSchedule }} starting {{ repaymentDate }}.',
+        { claimantName: claimantName, amount: amount, paymentSchedule: paymentSchedule, repaymentDate: repaymentDate }) }} </p>
+    <p>
+      {{ t('We’ll contact you when they respond.') }}
+    </p>
+  {% endif %}
+  <p>
+    {{ _downloadResponseLink(claim) }}
+  </p>
+{% endmacro %}
+
+{% macro claimFullAdmissionForClaimantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('The defendant’s response') }}</h2>
+
+  {% set claimantName = claim.claimData.claimant.name %}
+  {% set defendantName = claim.claimData.defendant.name %}
+  {% set amount = claim.totalAmountTillToday | numeral %}
+
+  {% if claim.response.paymentIntention.paymentDate %}
+    {% set paymentDate = claim.response.paymentIntention.paymentDate | date %}
+  {% endif %}
+
+  {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+    <h2 class="heading-small">{{ t('The defendant will pay you immediately') }}</h2>
+    <p> {{ t('The defendant said they’ll pay you immediately. They must make sure you have the money by 4pm on {{ paymentDate }}.',
+        { paymentDate: paymentDate }) }} </p>
+    <h2 class="heading-small"> {{ t('If the defendant pays you') }} </h2>
+    <p> {{ t('You need to email {{ supportMail }} to let us know. Include the date you were paid.', {supportMail: supportEmailAddress }) }} </p>
+    {% if claim.response.paymentIntention.isPastPaymentDeadline %}
+      <h2 class="heading-small">{{ t('What to do if they haven’t paid') }}</h2>
+      <p> {{ t('If the defendant doesn’t pay you, you can request a County Court Judgment.') }} </p>
+    {% else %}
+      <h2 class="heading-small">{{ t('What to do if they don’t pay') }}</h2>
+      <p> {{ t('If the defendant has not paid you by {{ paymentDate }}, you can request a County Court Judgment.', {paymentDate: paymentDate}) }} </p>
+    {% endif %}
+  {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
+    <p> {{ t('{{ defendantName }} admits they owe all the money. They’ve offered to pay you in full by {{ paymentDate }}.',
+        { defendantName: defendantName, paymentDate: paymentDate }) }} </p>
+    <p>
+      {{ t('You can accept or reject their offer.') }}
+    </p>
+    {{ _viewAndRespondOffer(claim) }}
+  {% elseif  claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
+    <p> {{ t('{{ defendantName }} admits they owe all the money. They’ve offered to pay you in instalments.',
+        { defendantName: defendantName }) }} </p>
+    <p>
+      {{ t('You can accept or reject their offer.') }}
+    </p>
+    {{ _viewAndRespondOffer(claim) }}
+  {% endif %}
+{% endmacro %}
+
+{% macro claimFullAdmissionForClaimantDashboard(claim) %}
```

Comment:
Can you move this macro above `claimFullAdmissionForClaimantClaimDetails` so that we have dashboard macro first then details macro. It is easier to read IMO.